src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1661,7 +1661,7 @@ void TorrentHandle::handleTrackerErrorAlert(const lt::tracker_error_alert *p)
     {
```

Comment:
Please also remove this part from the comment above: "TrackerEntry::isWorking() returns true if at least one endpoint works."

Reply:
At line 1657-1658 if you can't find it.